tests/unit/spec/base/mozilla-traffic-cop.js

```diff
@@ -40,6 +40,14 @@ describe('mozilla-traffic-cop.js', function () {
             cop.init();
             expect(cop.verifyConfig).toHaveBeenCalled();
         });
+
+        it('should initialize if DNT helper function is not available', function() {
+            var dntBak = window._dntEnabled;
+            window._dntEnabled = false;
```

Comment:
Yep, that'll definitely work, and matches what we want to test better than setting to `false`.